packages/sources/mock-ea/src/endpoint/price.ts

```diff
@@ -0,0 +1,45 @@
+import { Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ExtendedConfig } from '../config'
+
+export const supportedEndpoints = ['price']
+
+export interface ResponseSchema {
+  data: {
+    price: number
+  }
+}
+
+export const inputParameters: InputParameters = {}
+
+interface ResponseInfo {
+  result: number
+  lastUpdated?: number
+}
+
+const responseInfo: ResponseInfo = {
+  result: 1000,
+}
+
+export const execute: ExecuteWithConfig<ExtendedConfig> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const lastUpdated = responseInfo.lastUpdated
```

Comment:
Will this ever be set?

Reply:
But it won't ever be set _before_ the `if` statement, right? Just wondering if this check actually does something since it's not set before we're inside the if statement.

Reply:
so this check will run when a request comes in and it'll get set for the first time after the first request.  Subsequent requests then check against it and update the result accordingly.  This seems to work when as the results do vary when I tried it locally.

Reply:
Ooooh because it's a global variable -- I see now

Reply:
yup exactly